ops/gauntlet/gauntlet_plus_plus_starknet.go

```diff
@@ -0,0 +1,313 @@
+package gauntlet
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"context"
+	"github.com/rs/zerolog/log"
+	g "github.com/smartcontractkit/gauntlet-plus-plus/sdks/go-gauntlet/client"
+)
+
+var (
+	sgpp *StarknetGauntletPlusPlus
+)
+
+type Request struct {
+	Input map[string]any 		 `json:"input"`
+	Command string					 `json:"command"`
+}
+
+type StarknetGauntletPlusPlus struct {
+	client				*g.ClientWithResponses
+	gr      			*http.Response
```

Comment:
what's `gr` stand for and where is it used? is it the acronym for "gauntlet response"?

Reply:
Yup. Do you think I should expand it to `response`?